### DISCUSSION TITLE
Saving song info for .m4a files

### DISCUSSION THREAD

**Discussion by alkaliphile** [audacious-media-player/audacious#1460] [Q&A] [answered]
I'm trying to edit the song info for some m4a files. Adding Title, Artist, etc. I see the following errors about these read_tag() and write_tag() functions not being supported, though.

Is there a way to accomplish this without booting into Windows and doing it with Winamp?

INFO audtag.cc:35 [read_tag]: read_tag() not supported for file:///home/c/Music/song.m4a

INFO ../src/libaudcore/vfs.cc:105 [VFSFile]: <0x5616c30e5e20> open (mode r+) file:///home/c/Music/song.m4a
INFO audtag.cc:48 [write_tuple]: write_tag() not supported for file:///home/c/Music/song.m4a

**Comment by radioactiveman** [comment, marked as answer]
> Is there a way to accomplish this without booting into Windows and doing it with Winamp?

On Linux you can use applications like [EasyTAG](https://wiki.gnome.org/Apps/EasyTAG) or [tag editor](https://github.com/Martchus/tageditor) which support more tag formats than Audacious.

**Reply by SirGeoffreyB** [reply]
[EasyTAG](https://wiki.gnome.org/Apps/EasyTAG) & [tag editor](https://github.com/Martchus/tageditor) sound like they are probably better/more advanced, but [Audacity](https://www.audacityteam.org/) will do some tagging as well. A lot of people have that already installed.

**Comment by alkaliphile** [comment]
tag editor works great. thanks for the suggestions!
